src/EditorFeatures/CSharpTest/Diagnostics/UseAutoProperty/UseAutoPropertyTests.cs

```diff
@@ -1000,5 +1000,64 @@ int Q
     int Q { get; }
 }", fixAllActionEquivalenceKey: FeaturesResources.Use_auto_property);
         }
+
+        [WorkItem(23735, "https://github.com/dotnet/roslyn/issues/23735")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        public async Task ExplicitInterfaceImplementationGetterOnly()
+        {
+            await TestMissingInRegularAndScriptAsync(@"
+namespace RoslynSandbox
```

Comment:
Small nit.  When making test cases, try to make them as small as possible (i'll admit we don't always follow that).  So, remove things like unnecessary namespaces as they're not relevant to the fix.

Note: you don't have to change this.  just something to keep in mind for the future.